rust/private/repository_utils.bzl

```
@@ -235,6 +236,9 @@ def BUILD_for_rust_toolchain(
     rustc_srcs = "None"
     if include_rustc_srcs:
         rustc_srcs = "\"@{workspace_name}//lib/rustlib/src:rustc_srcs\"".format(workspace_name = workspace_name)
+    rustfmt_label = "None"
+    if rustfmt_version:
+        rustfmt_label = "@{workspace_name}//:rustfmt_bin".format(workspace_name = workspace_name)
```

You'll need quotes here if you want to be able to pass `None` in the template in this way.
```suggestion
        rustfmt_label = "\"@{workspace_name}//:rustfmt_bin\"".format(workspace_name = workspace_name)
```